fizz-buzz-streams/src/test/java/com/pubsap/eng/streams/FizzBuzzAggregatorTest.java

```diff
@@ -0,0 +1,191 @@
+package com.pubsap.eng.streams;
+
+import com.pubsap.eng.schema.*;
+
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import io.confluent.kafka.schemaregistry.client.MockSchemaRegistryClient;
+import io.confluent.kafka.schemaregistry.client.rest.exceptions.RestClientException;
+import io.confluent.kafka.streams.serdes.avro.SpecificAvroSerde;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.streams.*;
+import org.apache.kafka.streams.kstream.*;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.time.Instant;
+import java.util.*;
+
+import static org.apache.kafka.common.utils.Utils.mkEntry;
+import static org.apache.kafka.common.utils.Utils.mkMap;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class FizzBuzzAggregatorTest {
+
+    private static TopologyTestDriver testDriver;
+    private static TestInputTopic<InputKey, Item> inputTopic;
+    private static TestOutputTopic<OutputKey, Output> outputTopic;
+    final private static String srConfigKey = "schema.registry.url";
+
+    @BeforeAll
+    public static void setUp() throws IOException, RestClientException {
+
+        final Config config = ConfigFactory.load();
+
+        Properties properties = new Properties();
+        properties.put(StreamsConfig.BOOTSTRAP_SERVERS_CONFIG, config.getString("bootstrap.servers"));
+        properties.put(StreamsConfig.APPLICATION_ID_CONFIG, config.getString("application.id"));
+        properties.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "latest");
+        properties.put(StreamsConfig.CACHE_MAX_BYTES_BUFFERING_CONFIG, 0);
+
+        String inputTopicName = config.getString("aggregator.input.topic.name");
```

Comment:
I'd like to leave this to IT tests (same for `FizzBuzzCompleteTopology`). Just take a constant value at line 89/90.